### PR TITLE
Changed the file Unittest and added file Helper

### PR DESCRIPTION
**Pull request by Volha-bratuhin**
Please check and approve my files

### CODE REVIEW

File: 02_Front_end_testing/VBratuhin_Rideshare_Unittest.py
Comment (SergioUS) [inline 02_Front_end_testing/VBratuhin_Rideshare_Unittest.py:1]: @Volha-bratuhin pls double check all deletions and changes in this file. If you confirm that's OK, I'll do approve this PR 👍

Comment (Volha-bratuhin) [reply 02_Front_end_testing/VBratuhin_Rideshare_Unittest.py:1]: I checked everything again. Everything is good. Thank you.

Comment (SergioUS) [reply 02_Front_end_testing/VBratuhin_Rideshare_Unittest.py:1]: Ok @Volha-bratuhin I'll do approve this PR 👍 @Andrei-Markovich @Timura2811 @TatsianaLentz @ZaraAvramov pls do review as well to make sure this script is OK for you.

**Review by SergioUS — APPROVED** [review]
LGTM! @Volha-bratuhin 👍

**Pull request merged**